### PR TITLE
Vet clinic database: add 'join table' for visits

### PR DESCRIPTION
**Pull request by Fombi-Favour**
In this activity, we:
- created `vets` table.
- created `specializations` and `visits` join table.
- added data into the tables
- filtered data with queries

## Queries
- >Who was the last animal seen by William Tatcher?
project4-sc1
- >How many different animals did Stephanie Mendez see?
project4-sc2
- >List all vets and their specialties, including vets with no specialties.
project4-sc3
- >List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
project4-sc4
- >What animal has the most visits to vets?
project4-sc5
- >Who was Maisy Smith's first visit?
project4-sc6
- >Details for most recent visit: animal information, vet information, and date of visit.
project4-sc7
- >How many visits were with a vet that did not specialize in that animal's species?
project4-sc8
- >What specialty should Maisy Smith consider getting? Look for the species she gets the most.
project4-sc9

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -32,4 +32,41 @@ UPDATE animals SET owner_id=(SELECT id from owners WHERE full_name='Sam Smith')
...
 UPDATE animals SET owner_id=(SELECT id from owners WHERE full_name='Melody Pond') WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
-UPDATE animals SET owner_id=(SELECT id from owners WHERE full_name='Dean Winchester') WHERE name IN ('Angemon', 'Boarmon');
\ No newline at end of file
+UPDATE animals SET owner_id=(SELECT id from owners WHERE full_name='Dean Winchester') WHERE name IN ('Angemon', 'Boarmon');
+
+-- inserting data in "vets" table
+INSERT INTO vets (name, date_of_graduation, age) VALUES
+('William Tatcher', '04-23-2000', 45),
+('Maisy Smith', '01-17-2019', 26),
+('Stephanie Mendez', '05-04-1981', 46),
```
Comment (franclobo) [inline data.sql]: - [ ] Please correct the age to have the correct data 👇🏽 

> Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Fombi-Favour, 

# Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct schema ✔️ 
- Correct queries ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @Fombi-Favour 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Implemented suggested changes
-  ✅ Great work using the right Gitflow 
-  ✅ Your README looks good

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Efiamotu-1 — COMMENTED** [review]
Hi @Fombi-Favour , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

image

## Highlights :dart: 
* Correct GitHub flow or Gitflow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**